app.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "foreign-travel-content-checker",
+  "repository": "https://github.com/alphagov/foreign-travel-content-checker",
+  "env": {
+    "GOVUK_APP_DOMAIN": {
+      "value": "www.gov.uk"
+    },
+    "GOVUK_WEBSITE_ROOT": {
+      "value": "https://www.gov.uk"
+    },
+    "PLEK_SERVICE_CONTENT_STORE_URI": {
+      "value": "https://www.gov.uk/api"
+    },
+    "PLEK_SERVICE_SEARCH_URI": {
+      "value": "https://www.gov.uk/api"
+    },
+    "PLEK_SERVICE_STATIC_URI": {
+      "value": "assets.digital.cabinet-office.gov.uk"
+    },
```

Comment:
Do you know if you need all these env vars? they seem the type that are needed for GOV.UK dependencies that I doubt this app has.

Reply:
Nope, the app doesn't need any of these. Just a copy-and-paste issue as I wasn't sure. Fixed.